### PR TITLE
Update dapr version

### PR DESCRIPTION
**Pull request by jkotalik**
Fixes https://github.com/dotnet/tye/issues/285.

Having trouble running this locally at the moment due to the error:

```
"failed to start metrics server: listen tcp :49398: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.
```

### CODE REVIEW

**Comment by jkotalik** [comment]
Figured it out, we had a bad check for the service name. This is working now.

File: src/Microsoft.Tye.Hosting/Model/Application.cs [hunk 1]
```diff
@@ -38,6 +38,10 @@ public void PopulateEnvironment(Service service, Action<string, string> set, str
                 // Inject normal configuration
                 foreach (var pair in service.Description.Configuration)
                 {
+                    if (pair.Name == "METRICS_PORT")
+                    {
+                        Console.WriteLine("wow");
+                    }
```
Comment (rynowak) [inline src/Microsoft.Tye.Hosting/Model/Application.cs:44]: uhhhhh

Comment (rynowak) [reply src/Microsoft.Tye.Hosting/Model/Application.cs:44]: wow such metrics.

Comment (jkotalik) [reply src/Microsoft.Tye.Hosting/Model/Application.cs:44]: How did this get here????? 😉

**Pull request merged**